webapp/angular.json

```diff
@@ -90,7 +90,7 @@
         "test": {
           "builder": "@angular-devkit/build-angular:karma",
           "options": {
-            "main": "tests/karma/ts/test.ts",
+            "main": "tests/karma/test.ts",
```

Comment:
The `test.ts` in this level will recognise specs from `/js` and `/ts` directories